index.js

```diff
@@ -116,6 +116,16 @@ alfy.fetch = async (url, options) => {
 		...options,
 	};
 
+	// Set responseType to 'text' if options.json === false or default to Json
+	if ('json' in options && options.json === false) {
+		delete options.json;
+		options.responseType = 'text';
+		options.resolveBodyOnly = true;
+	} else {
+		options.responseType = 'json';
+		options.resolveBodyOnly = true;
```

Comment:
It's kind moot to set this twice. Just put it on line 117.

Reply:
I wanted to be explicit about where the `resolveBodyOnly` comes from / and what for, but I see your POV as well. Sec

Reply:
I looked at the code again and realised that setting these options pre-caching was needlessly polluting the cache keys (suddenly, every cache key would also include the "resolveBodyOnly" and "responseType"), which I realise was not what we needed. Pulled the resolveBodyOnly out of the IF and moved it post-caching